src/main.py

```
@@ -0,0 +1,78 @@
+import asyncio
+import os
+from kubernetes import client, config as k8s_config
+from websockets import connect
+from websockets.exceptions import WebSocketException
```

Ruff (0.8.2)

`websockets.exceptions.WebSocketException` imported but unused

Remove unused import: `websockets.exceptions.WebSocketException`

(F401)